web/extensions/core/openagi.js

```diff
@@ -4,12 +4,18 @@ const ext = {
 	name: "Comfy.OpenAGI",
 
 	async setup(){
-		const handleWorkflowEvents = async function (event) {
-			const parsedEventData = JSON.parse(event.data)
-			
 
-			if (parsedEventData.event === "loadWorkflow") {
-				// try{}catch(e)
+    window.parent.postMessage(
+      JSON.stringify({
+        event: "iframeLoaded", // Indicate that the iframe (ComfyUI) has been initialized
+      }),
+      "*" // Use "*" to allow communication regardless of origin
```

Comment:
this needs to be tested properly, because your are not receiving event from frontend and then responding, event is sent from comfy to frontend, to which origin it will send the event

Reply:
it will send to all orgin